Add pub.dev filter to Wasm page (#10758)
flutter/website: src/content/platform-integration/web/wasm.md

- [`package:web`][], which replaces `dart:html` (and other web libraries)
- [`dart:js_interop`][], which replaces `package:js` and `dart:js`

The Dart and Flutter teams have migrated most packages

> to be compatible with Wasm support in Flutter,

Reviewer: **suggestion:** Make this more concise.
```suggestion
to support Wasm in Flutter,
```

such as [`package:url_launcher`][].

> You can find packages that are compatible with Wasm
> by using the [`wasm-ready`][] filter on [pub.dev][].

Reviewer: **issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To find Wasm-compatible packages, 
use the [`wasm-ready`][] filter on [pub.dev][].
```
